src/inc/til/spsc.h

```diff
@@ -119,7 +119,7 @@ namespace til::spsc
             }
             else
             {
-                return static_cast<T*>(::operator new(size, std::align_val_t(alignment)));
+                return static_cast<T*>(::operator new(size, static_cast<std::align_val_t>(alignment)));

```

Comment:
These are the only ones I don't fully understand. Why are we adding `static_cast` to these?

Reply:
Better than a functional cast